apps/wallet-mobile/src/migrations/4_26_0.test.ts

```diff
@@ -64,15 +64,15 @@ describe('4_26_0 migrations', () => {
     expect(storage.setItem).toHaveBeenCalledTimes(3)
     expect(storage.setItem).toHaveBeenCalledWith('1', {
       ...pwdWalletMeta,
-      addressMode: 'multiple',
+      addressMode: 'single',
```

Comment:
this will migrate current wallets to `single` and the user will realize this change after opening the receive, just a heads up.